livereveal/main.js

```diff
@@ -17,8 +17,14 @@ var config_section = new configmod.ConfigSection('livereveal',
                             {base_url: utils.get_body_data("baseUrl")});
 config_section.load();
 var config = new configmod.ConfigWithDefaults(config_section, {
+    controls: true,
+    progress: true,
+    history: true,
+    width: 1140,
+    minScale: 1.0,
```

Comment:
This sould be not modificable because of some codemirror issue, so I don't want to expose it... please remove it